src/styles/blocks/cards-box.scss

```
@@ -3,12 +3,14 @@
 }
 
 .cards-box {
+  padding: 50px $paddings;
   width: $box-width;
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: space-between;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(auto-fill, $card-width);
```

```suggestion
  grid-template-columns: repeat(4, $card-width);
```
можно задать количество колонок как в задании и тогда не нужен width:  и медиа тоже можно будет удалить
@media (min-width: $box-width) {
    grid-template-columns: repeat($cards-in-line, $card-width);
  }

---

It's correct comment for this particular task.
But sometimes it is better not to lock nember of columns for adaptivity reasons.